rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala

```diff
@@ -130,7 +130,7 @@ object JsInitContextLinkUtil extends Loggable {
 
   def createGroupLink(id:NodeGroupId) = {
     roGroupRepository.getNodeGroup(id) match {
-      case Full((group,_)) => <span> {SHtml.a(() => S.redirectTo(groupLink(id)), Text(group.name))} (Rudder ID: {id.value})</span>
+      case Full((group,_)) => <span> <a href={baseGroupLink(id)}>{group.name}</a> (Rudder ID: {id.value})</span>
```

Comment:
I verified the other calls are ok!

I still don't know why it adding /rudder ...)